.github/workflows/tests-8.x.yaml

```
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: ['8.0', '8.1']
+        version: ['8.0', '8.1', '8.2', '8.3']

```

The latest version shouldn't be there, only the older ones:

```suggestion
        version: ['8.0', '8.1', '8.2']

```

---

Thank you for the hint. Fixed.